src/main/java/com/karlofduty/EMCTools/commands/CalcCommand.java

```diff
@@ -0,0 +1,80 @@
+package com.karlofduty.EMCTools.commands;
+
+import com.palmergames.bukkit.towny.object.TownyPermission;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import com.palmergames.bukkit.towny.utils.PlayerCacheUtil;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.BlockInventoryHolder;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.inventory.meta.EnchantmentStorageMeta;
+
+public class CalcCommand implements CommandExecutor {
+	@Override
+	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+		if (!(sender instanceof Player)) {
+			sender.sendMessage("§CCannot use this command as console.");
+			return true;
+		}
+
+		Player player = (Player) sender;
+		Block block = player.getTargetBlock(5);
+		if (block == null) return true;
+		if (!PlayerCacheUtil.getCachePermission(player, block.getLocation(), block.getType(), TownyPermission.ActionType.SWITCH)) {
+			sender.sendMessage("§CCannot use this command without permission.");
+			return true;
+		}
+		if (block.getState() instanceof BlockInventoryHolder) {
+			float sum = 0;
+			BlockInventoryHolder invhold = (BlockInventoryHolder) block.getState();
+			Inventory inv = invhold.getInventory();
+
+			for (ItemStack item : inv.getContents()) {
+				if (item == null) continue;
+				int amount = item.getAmount();
+
+				switch (item.getType()) {
+					case GOLD_INGOT: sum += amount; break;
+					case GOLD_BLOCK: sum += amount * 9; break;
+					case BEACON: sum += amount * 60; break;
+					case GOLDEN_APPLE: sum += amount * 8; break;
+					case DIAMOND_BLOCK: sum += amount * 0.15; break;
```

Comment:
What exactly would these values mean?

Reply:
amount of itemstack * price for one item

if item is not stackable, e.g. totem, its only sum += 300
btw, I might add a command for updating economy values

Reply:
Who decides the price for the items? Also this plugin is used on other servers of ours where Towny isn't a thing.

Reply:
what plugin is used on other server? this code I wrote is 100% mine. Yeah prices might be biased but it can be an average of prices taken from most popular markets et cetera. mods would be able to change them via cmd

Reply:
You don't expect me to tell you all the plugins we use?